### PR TITLE
Mo

### PR DESCRIPTION
**Pull request by coders-kitchen**

### CODE REVIEW

File: mobile-inbound-api/v1/openapi.yaml [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+    tyntec's application will retry for a maximum of 48 hours.
+
+  contact:
+    name: tyntec API Support
+    url: http://www.tyntec.com/support
+    email: support@tyntec.com
+  version: '1.0'
+  x-repository: https://github.com/tyntec/api-collection/blob/master/mobile-inboud-api
+  x-major-version: v1
+servers:
```
Comment (coders-kitchen) [inline mobile-inbound-api/v1/openapi.yaml]: I think this can be removed :)

File: mobile-inbound-api/v1/openapi.yaml [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+  - url: https://{defaultHost}
+    variables:
+      defaultHost:
+        default: www.example.com
+paths:
+components:
+  schemas:
```
Comment (coders-kitchen) [inline mobile-inbound-api/v1/openapi.yaml]: In general I miss examples eg:

        from:
          type: string
          example: 49176244234

File: mobile-inbound-api/v1/openapi.yaml [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+        originMCC:
+          type: string
+          description: A representative MCC (Mobile Country Code) of the originating network.
+        originMNC:
+          type: string
+          description: A representative MNC (Mobile Network Code) of the originating network.
+        originTtId:
+          type: string
+          description: The tyntec id of the operator.
+            The mapping can be found in the customer lounge under   “home>resource center>network lists>mobile outbound sms”   (https://customerlounge.tyntec.com/resource-centre.html).
```
Comment (coders-kitchen) [inline mobile-inbound-api/v1/openapi.yaml]: I think this link needs to be changed ;)

File: mobile-inbound-api/v1/example.json [hunk 4]
```diff
@@ -0,0 +1,22 @@
+{
```
Comment (coders-kitchen) [inline mobile-inbound-api/v1/example.json]: ❤️

File: mobile-inbound-api/v1/openapi.yaml (hunk 1 above)
Comment (oneiros-de) [reply mobile-inbound-api/v1/openapi.yaml]: Why? It's copied more or less verbatim from the chat-api.

Comment (coders-kitchen) [reply mobile-inbound-api/v1/openapi.yaml]: it's about the "servers:" :)

**Pull request merged**